packages/java-parser/src/index.js

```diff
@@ -24,6 +29,13 @@ function parse(inputText, entryPoint = "compilationUnit") {
 
   parser.input = lexResult.tokens;
 
+  // prettier-ignore support
```

Comment:
I think you accidentally disabled formatting on this next line 😄

Reply:
woops !